core/image/common.odin

```diff
@@ -1131,4 +1131,4 @@ write_bytes :: proc(buf: ^bytes.Buffer, data: []u8) -> (err: compress.General_Er
 		return compress.General_Error.Resize_Failed
 	}
 	return nil
-}
\ No newline at end of file
+}
```

Comment:
Not sure what this meaningless change to this file is doing in the patch?

Reply:
Sublime automatically inserted newline :/